readme.md

```diff
@@ -1,30 +1,261 @@
-# Usage
+# @bitnoi.se/scheduler
 
-In order to display Scheduler properly it's required to import styles:
+Typescript oriented, light-weight and ultra fast React Component for creating a gantt charts.
+
+### Installation
+
+```bash
+# yarn
+yarn add @bitnoi.se/scheduler
+# npm
+npm install @bitnoi.se/scheduler
+```
+
+### Example Usage
+
+1. import required styles for scheduler
```

Comment:
Step 2 has Scheduler (upper case) - we should unify this. I think we should go with S.